src/SymfonyCache/KernelDispatcher.php

```diff
@@ -96,6 +97,10 @@ public function flush()
 
         $httpCache = $this->httpCacheProvider->getHttpCache();
 
+        if (null === $httpCache) {
+            throw new ProxyUnreachableException('Cannot reach Symfony HttpCache as the Kernel does not know about it.');
```

Comment:
could we add something more here to help with the most common setup error? "Kernel did not return a HttpCache instance. Did you forget $kernel->setHttpCache($cacheKernel) in your front controller?"

Reply:
Good hint, done :)